### DISCUSSION TITLE
Wrong Chrome version?

### DISCUSSION THREAD

**Discussion by Neith343** [Discord-OTP-Forcer/Discord-OTP-Forcer#128] [Community Support]
![Screenshot 2024-07-21 225616](https://github.com/user-attachments/assets/9e4cc7f7-1423-49d9-bca9-405a63d6bab7)

Trying to run the start.bat file, and when I do, I get this message at the very bottom. I'm not sure what I did wrong, or if it's the "ungoogled-chromium" thing I have to install still, but I need help.

**Comment by Neith343** [comment]
This was to help a friend. They didn't get my same error message, but they managed to fix their problem and are now brute forcing their 2fA code. Thank you for developing this, it truly is a remarkable thing.

**Comment by Kikukarenai** [comment]
I'd like to know aswell.

**Reply by Inbydev** [reply]
@Kikukarenai  could you try with the latest commit?

https://codeberg.org/Discord-OTP-Forcer/Discord-OTP-Forcer/releases/tag/v0.5.0-beta

**Comment by barisdx** [comment]
Can someone please share a solution if they have one? Until yesterday it was working just fine and now I'm getting the same error except it's 117 instead of 122. The funny thing is I've made sure both Chrome and chromedriver versions are the same and 127, still the same error. I'm thinking that the script fails to detect the version of the driver for some reason? 

This is our only hope of getting our Discord accounts back, so let's please brainstorm any idea.

**Reply by barisdx** [reply]
Yes, it does. Now I've installed Chromium but it only triggered a different issue.
![image](https://github.com/user-attachments/assets/06631abb-a5c5-4384-84d0-f2d2592d9166)

**Reply by barisdx** [reply]
Update: Deleted Chromium and did what @progressEdd suggested in #132 and it seems to be working now.

**Reply by Inbydev** [reply]
@barisdx  could you try with the latest commit?

https://codeberg.org/Discord-OTP-Forcer/Discord-OTP-Forcer/releases/tag/v0.5.0-beta

**Comment by progressEdd** [comment]
I had this issue on Fedora Linux, and I solved it by installing chromium. I can't say I've tested it on windows, but installing chromium resolved my issue.

Alternatively, we can switch the requirement from webdriver_manager to chromedriver-autoinstaller. I started a pr that has a patch and updates needed. This solution doesn't require the user to have chromium installed. Please pull and test if it works on other platforms without needing to install chrome or chromium
https://github.com/Derpitron/Discord-OTP-Forcer/pull/132

**Comment by Inbydev** [comment]
@Neith343 could you try with the latest commit?

https://codeberg.org/Discord-OTP-Forcer/Discord-OTP-Forcer/releases/tag/v0.5.0-beta
